IVAN-MARAKHOVSKII/index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="../node_modules/normalize.css/normalize.css" />
+    <link rel="stylesheet" href="style.css" />
+    <title>New Jersey</title>
+    <link
+      href="https://fonts.googleapis.com/css?family=Source+Sans+Pro&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <div class="grid">
+      <div class="wheatherWidget">
+        <div class="refreshIcon">
+          <img src="refresh.png" alt="" />
+        </div>
+        <div class="wheatherImage">
+          <img src="cloud_icon.png" />
+        </div>
+        <div class="infoBlock">
+          <img src="tempIcon.png" class="tempIcon" />
+          <div class="infoText">
+            <span class="infoTextTop">Cloudy Skies</span><br /><span
```

Comment:
Since it's not address, and the first part of the text is related to the weather and another to the location, I'd suggest to avoid using `br` in such cases, the common use case for `br` is full address with street / apartment / city etc, also it will become easier to style